types/transaction.go

```diff
@@ -123,6 +123,21 @@ type TxData interface {
 	copy() TxData
 }
 
+func (tx *Transaction) String() string {
+	recipient := ""
```

Comment:
I think you don't need this convert to string, and what not. In the `sprintf` just use `txn.To()`, it will write `<nil>` if `To` is not defined, and if it is, it will automatically call `.String()` function on it.

Reply:
Fixed https://github.com/Ethernal-Tech/blade/pull/195/commits/79c0e49d286636fca7bec125db980ace70d842f5